README.md

```diff
@@ -184,6 +184,11 @@ vim.keymap.set(
   :RustLsp[!] debuggables {args[]}?
   :RustLsp[!] debug {args[]}?
   ```
+  Example:
+  1. Press escape
+  2. `:RustLsp debuggables`
+  3. You will be prompted with choice, for example select 1.
```

Comment:
This is redundant and would have to be added for any command that can prompt (there are many) for consistency.